packages/server/app/Controllers/Http/SearchUsersController.ts

```diff
@@ -138,6 +186,83 @@ export default class SearchUsersController {
         };
     }
 
+    public async listUserFollowers({
+        request,
+    }: HttpContextContract): Promise<ListUserFollowersResponseBody> {
+        const rawBody = request.body();
+        const { page, searchQuery, userID, tmpAuthUserID } =
+            ListUserFollowersRequestBody.parse(rawBody);
+
+        //Checking relations visibility
+        await throwErrorIfRequestingUserCanNotAccessRelatedUserRelationsVisibility(
+            {
+                relatedUserID: userID,
+                requestingUserID: tmpAuthUserID,
+            },
+        );
+        //
+        const usersFollowersPagination = await User.query()
+            .whereNot('uuid', userID)
+            .andWhere('nickname', 'ilike', `${searchQuery}%`)
+            .andWhereHas('following', (userQuery) => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏To get a user's followers, you get all the users that follow her and are not her. One's brain must be active to understand this relation.